.github/workflows/release.yml

```diff
@@ -46,8 +46,7 @@ jobs:
         with:
           draft: false
           files: |
-            webgoat-server/target/webgoat-server-${{ env.WEBGOAT_MAVEN_VERSION }}.jar
-            webwolf/target/webwolf-${{ env.WEBGOAT_MAVEN_VERSION }}.jar
+            webgoat/target/webgoat-${{ env.WEBGOAT_MAVEN_VERSION }}.jar
```

Comment:
The docker build and push parts need to be adjusted to the refactored standards.

Reply:
think I already fixed this, I cannot find webwolf anymore in the `release.yml`

Reply:
I did not mean this line. But the "Build and Push" part of the release.yml. That still refers to docker/Dockerfile and web goat/goatandwolf
That needs to be changed to the root location of the Docker file and the tag name as mentioned in the Readme

Reply:
fixed